backend/backend/api/routes/player.py

```diff
@@ -0,0 +1,26 @@
+import uuid
+from typing import Annotated
+
+from fastapi import APIRouter, Depends, status
+
+from backend.api.models.player import Player
+from backend.api.dependencies.player import players, get_player_by_id
+
+router = APIRouter(tags=["player"])
+
+
+@router.post("/players/{player_name}",
```

Comment:
I didn't noticed that you provide `player_name` here via path and not body

Reply:
that is becuase it's an integral part of player creation?

Reply:
you cant tell me to use path parameters without json in one cr and then change your mind mare hours in another cr- that's just f*cked up

Reply:
Firstly, I can, you don't have to agree with me.
Secondly, you are being not nice

We do have PRD that shows what suppose to be the API, if you fill a contradiction, you can point your finger towards it and tell me that I'm wrong.

Lastly, POST is meant for creation of a resource the server doesn't have, I can't tell you to create something you don't know about, luckly fastapi solves this issues with [this feature](https://fastapi.tiangolo.com/tutorial/body-multiple-params/#singular-values-in-body)

Don't lash on me when I'm CRing u, it is not fair nor nice and I don't want to work under such hostility

Reply:
you are being inconsistent in your CR - sayung yo use path parameters and then body parameters is inconsistent- and I will not buy into that loop of changes

Reply:
it doesn't match the PRD, fix please

Reply:
no